package.json

```diff
@@ -22,7 +22,7 @@
   },
   "main": "electron/main.mjs",
   "dependencies": {
-    "@gptscript-ai/gptscript": "^0.9.5-rc4",
+    "@gptscript-ai/gptscript": "github:gptscript-ai/node-gptscript#0ec49d9df04627afe4518f4689dc212c0d03cdd6",
```

Comment:
I had to fix an issue in `node-gptscript` to get the `respondingTool` stuff to work correctly.